jetpack-io/devbox
# VSCode extension for devbox

## Summary
Added vscode extension for devbox under `vscode-extension/`.
It shows 15 file changes but most of the implementation is in `extension.ts` and `package.json` files.

## How was it tested?
- cd vscode-extension/
- yarn install
- open vscode in this directory
- press F5 or press run in the run & debug section from the sidebar in vscode
- in the new opened vscode window, open a devbox project then open the vscode integrated Terminal
- confirm terminal runs devbox shell automatically
